src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -132,11 +128,13 @@ public void setpLossofHet(final double pLossofHet) {
      */
     public Map<String,Fingerprint> loadFingerprints(final File fingerprintFile, final String specificSample) {
         final IntervalList loci = this.haplotypes.getIntervalList();
-        final GenotypeReader reader = new GenotypeReader();
-        final GenotypeReader.VariantIterator iterator = reader.read(fingerprintFile, loci);
+        final VCFFileReader read = new VCFFileReader(fingerprintFile);
```

Comment:
`read` is not a good name for an object which _reads_ variants.  I suggested `reader`, `vcfReader`, `in`, etc.